### PR TITLE
DB-3178: Post Detail Pages

### PR DESCRIPTION
**Pull request by enZane**
## What changes were made?
- Added the component Post that renders a single post
- In the footer, there is the /post prefix to fix the link and go to the post
- In the PostGridItem, I do the same
- Route created for /posts/[slug] that get the post by SSR
- Fixed a typo on drupal starter, in the Article component for the link class

## Where were the changes made?
- next-drupal-starter
- next-wordpress-starter

## How have the changes been tested?
- Locally and built

## Additional information

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

File: starters/next-wordpress-starter/pages/posts/[slug].js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { NextSeo } from "next-seo";
+import Layout from "../../components/layout";
+import Post from "../../components/post";
+
+import { getFooterMenu } from "../../lib/Menus";
+import { getPostBySlug } from "../../lib/Posts";
+
+export default function PageListTemplate({ menuItems, post }) {
```
Comment (CobyPear) [inline starters/next-wordpress-starter/pages/posts/[slug].js:8]: Rename to `PostListTemplate`

**Review by CobyPear — CHANGES_REQUESTED** [review]
@enZane nice job again! Just some small naming stuff and a suggestion for the changelog note in the changeset.

File: starters/next-wordpress-starter/components/Post.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export default function Post({
+  post: { title, date, featuredImage, content },
+}) {
+  return (
+    <article className="prose lg:prose-xl mt-10 mx-auto">
+      <h1>{title}</h1>
+      <p className="text-sm text-gray-600">{date}</p>
```
Comment (CobyPear) [inline starters/next-wordpress-starter/components/Post.js:10]: One more small suggestion here:
```suggestion
      <p className="text-sm text-gray-600">{new Date(date).toDateString()}</p>
```

**Review by CobyPear — CHANGES_REQUESTED** [review]
One more change I missed the first time. This will be good to go after that!

File: starters/next-wordpress-starter/components/Post.js (hunk 2 above)
Comment (enZane) [reply starters/next-wordpress-starter/components/Post.js:10]: Oh, thank you! I was thinking about if I need to format it but forgot to ask :p

**Review by CobyPear — APPROVED** [review]

**Pull request merged**